packages/frontend-api/src/Resources/config/graphql-types/ModelType/Transport/TransportDecorator.types.yaml

```diff
@@ -38,6 +38,6 @@ TransportDecorator:
             payments:
                 type: "[Payment!]!"
                 description: "List of assigned payments"
-            transportType:
-                type: "TransportType!"
-                description: "Type of transport"
+            transportTypeCode:
+                type: "String!"
```

Comment:
I would rather create the `TransportTypeEnum` + `Decorator` here, see eg https://github.com/shopsys/shopsys/blob/16.0/packages/frontend-api/src/Resources/config/graphql-types/EnumType/LoginTypeEnumDecorator.types.yaml

Reply:
🫡 fixed https://github.com/shopsys/shopsys/pull/3431/commits/494bdea7928dddc3d677968062b75462f2905b9f and https://github.com/shopsys/shopsys/pull/3431/commits/a1cd5838bd7c69ef0b65a83651843b7e8431a15c